src/helpers.js

```diff
@@ -44,6 +44,18 @@ export function getSwitch(path, {children, basePath}) {
   })[0] || null;
 }
 
+export function getActivePath(currentPath, basePath, currentSwitch) {
+  if (!currentSwitch) { return null; }
```

Comment:
I am fine either way, `null` feels more explicit, but `undefined` is more like it wasn't set ever.  Hmm...